### ISSUE TITLE
[Bug]: Incompatibility with ex nihilo sequentia unable to insert liquids in barrel

### ISSUE THREAD

**Issue by TimeheroTH** [cech12/WoodenBucket#3]
### Minecraft Version

1.18.2

### Forge Version

40.1.23

### Mod Version

1.0.2.1

### Modpack or other Mod

unable to place liquids in barrel of ex nihilo sequentia mod version: ExNihiloSequentia-1.18.2-20220530-183458.jar

### Describe the Issue

when i pick up a liquid from an exnihilo barrel the liquids could be taken correctly but when i want to put a liquid from the wooden bucket to the exnihilo barrel doesn't work; the liquid stay stucked in the bucket and don't fill the barrel

**Comment by cech12** [comment]
Thanks for your report! When I have time, I will look into this :)

**Comment by TimeheroTH** [comment]
Thank you i hope you could do as soon as you can because i have tried ceramic buckets and i have the same issue ;)

**Comment by cech12** [comment]
Yes, it should be an compatibility issue between BucketLib and Ex Nihilo. Maybe on Ex Nihilo side. ^^ But I will look into it in some days :)
